src/components/list/list.service.js

```diff
@@ -63,6 +63,45 @@ module.exports.deleteList = async (listId, accountId) => {
     }
 };
 
+/**
+ * Delete Item from List
+ * @param {String} listId list id
+ * @param {String} mediaType mediaItem type, movie or series
+ * @param {String} mediaId mediaItem id, mediaId
+ * @param {String} accountId Account identifier.
+ * @throws {Unauthorized} List doesnt exist or the user doesnt have permissions over it
+ * @throws {InternalServerError} Unhandled error.
+ */
+module.exports.deleteItemFromList = async (listId, mediaType, mediaId, accountId) => {
+    try {
+        var list = await listModel.findById(listId);
+
+        if (!list || list.accountId.toString() !== accountId)
+            throw new error.Unauthorized('The specified list doesnt exist or you dont have permissions over it');
+
+        list.mediaItems = DeleteItemFromMediaItems(list.mediaItems, mediaType, mediaId);
+
+        await listModel.findOneAndUpdate({ _id: listId }, list);
```

Comment:
You could do this instead: `await list.save()` since the method `findById` tracks the changes made to an object. More info: https://mongoosejs.com/docs/documents.html#updating

Reply:
Im not understanding this completely, but is it working properly?

In the meantime i will head to issue 27 just in case it could be a priority.

Reply:
Yup, I just forgot to merge it. done!